Makefile

```diff
@@ -1,10 +1,63 @@
-# Image URL to use all building/pushing image targets
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ROOT_DIR_RELATIVE := .
+
+include $(ROOT_DIR_RELATIVE)/common.mk
```

Comment:
Why do we need this?

Reply:
It contains common functionality. Eg: The help menu

Reply:
1). Could we use `REPO_ROOT` here instead of `ROOT_DIR_RELATIVE`?

2). In changing to use the more standard REPO_ROOT, what is the reason to change to the `git rev-parse` method of finding the root directory?
` export REPO_ROOT := $(shell git rev-parse --show-toplevel)`

We used to use:
`export PROJECT_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))`
which utilizes the Makefile's own location and the `make` command's built in list.

And it looks like VSphere and Azure Use:
`ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))`

Reply:
> It contains common functionality. Eg: The help menu

Yeesh... Our Makefile's grown that much hasn't it. Haha